pandas/io/formats/style.py

```diff
@@ -922,14 +953,18 @@ def set_td_classes(self, classes: DataFrame) -> Styler:
         ...     columns=[["level0", "level0"], ["level1a", "level1b"]])
         >>> classes = pd.DataFrame(["min-val"], index=["a"],
         ...     columns=[["level0"],["level1a"]])
-        >>> df.style.set_td_classes(classes)
+        >>> df.style.set_td_classes(classes)  # doctest: +SKIP
+                            level0
+                level1a	    level1b
+        a	    1	        2
+        b	    3	        4
```

Comment:
output unnecessary, and also I don't think this will render aligned (could be wrong)